src/main/java/com/savvato/tribeapp/services/ConnectServiceImpl.java

```diff
@@ -190,4 +185,27 @@ public boolean removeConnection(ConnectionRemovalRequest connectionRemovalReques
             return false;
         }
     }
+
+    @Override
+    public Boolean validateConnection(Long requestingUserId, Long toBeConnectedWithUserId) {
```

Comment:
The error messages here should include some relevant parameter information.. for instance the first error should read "The logged in user (USER_ID) does not match the issuing user (REQUESTING_USER_ID)." and likewise for the other log messages. This is so that when we have many many connections being made, we can better tell which error relates to the user IDs we happen to be testing with. It helps find signal in the noise.